### ISSUE TITLE
Symbols are not rendering 

### ISSUE THREAD

**Issue by thecalvinchan** [airbnb/react-sketchapp#351]
| -------------------------------------------------------------------------------------------------
| Reporting a bug or issue
| -------------------------------------------------------------------------------------------------

**Expected behavior:**

The react-sketchapp/examples/symbols project renders properly

**Observed behavior:**

It reports an error

```
/Users/calvinchan/react-sketchapp/examples/symbols  [git:master][calvinchan@ip-10-1-1-107] [15:03]
> yarn render
yarn run v1.6.0
$ skpm-build --watch --run
🖨  Copied src/manifest.json in 9ms
Error: Cannot render into Symbols page
line: 3109
sourceURL: /Users/calvinchan/react-sketchapp/examples/symbols/symbols.sketchplugin/Contents/Sketch/my-command.js
column: 16
stack: render@/Users/calvinchan/react-sketchapp/examples/symbols/symbols.sketchplugin/Contents/Sketch/my-command.js:3109:16
/Users/calvinchan/react-sketchapp/examples/symbols/symbols.sketchplugin/Contents/Sketch/my-command.js:4629:30
__skpm_run@/Users/calvinchan/react-sketchapp/examples/symbols/symbols.sketchplugin/Contents/Sketch/my-command.js:33077:17
__skpm_run@[native code]
🔩  Built ./my-command.js in 3249ms
```

**How to reproduce:**

clone repository and run `yarn render`

(on yarn 1.6)

**Sketch version:**

50.2

**Please attach screenshots, a zip file of your project, and/or a link to your github project**

**Comment by thecalvinchan** [comment]
This seemed to be an issue with the latest version on the npm registry (2.0.0)

**Comment by thecalvinchan** [comment]
looks like this issue occurs with v2.0.0 on npm, and not 1.1.0

**Comment by jaridmargolin** [comment]
Hi @thecalvinchan - Thanks for the report. I have reproduced the issue.

There are actually a few issues here:
1. The example attempts to render a `Document` into a `Page`. This should be removed from the example code.
2. The error is caused by rendering the `Document` into the `Page`, however it somehow doesn't get caught until `Error: Cannot render into Symbols page`. This should Error should get caught earlier in the render cycle, and thrown with an appropriate error message.

**Comment by thecalvinchan** [comment]
@jaridmargolin I don't think that the example is trying to render a `Document` (as defined by react-sketchapp) in the page. The default export defines a `const Document` that is just an `Artboard`. That document is rendered into the page.

Furthermore, 1.1.0 symbol overrides are not working as expected

**Comment by jaridmargolin** [comment]
@thecalvinchan - You are 100% right. How foolish of me to think a const named Document would refer to a Document 😂

In all honesty, this makes much more sense now. The Error is getting caught in the correct location. The error appears to be due to the fact that makeSymbol results in switching the current page.

I just created a small patch, #353, that should work around this issue.

**Comment by mathieudutour** [comment]
published in 2.1.0
